decoder/src/lib.rs

```diff
@@ -169,7 +171,7 @@ impl Table {
 
     fn get_with_level(&self, index: usize) -> Result<(Level, &str), ()> {
         let (lvl, format) = self._get(index)?;
-        Ok((lvl.ok_or(())?, format))
+        Ok((lvl.unwrap_or(Level::None), format))
```

Comment:
This is still an error when it happens

Reply:
Reverted to previous call to `lvl.ok_or(())?`.